### ISSUE TITLE
ASCII art replacements for the welcome banner

### ISSUE THREAD

**Issue by ghys** [openhab/openhab-distro#1139]
There goes one hour and a half 😅 I figured I'd give it a try and attempt to replace the ascii art for the Karaf console's banner, with something more resembling the official brand (or...?)
The current welcome banner, for reference:
```

                          __  _____    ____
  ____  ____  ___  ____  / / / /   |  / __ )
 / __ \/ __ \/ _ \/ __ \/ /_/ / /| | / __  |
/ /_/ / /_/ /  __/ / / / __  / ___ |/ /_/ /
\____/ .___/\___/_/ /_/_/ /_/_/  |_/_____/
    /_/                        3.0.0-SNAPSHOT
                               Build #1919

```
And some of the suggestions for a replacement:
```

    .-"""-. 
  .`       `.                       |   |,---.,--.
 /    .''.   \  ,---.,---.,---.,---.|___||___||__.
 ;  .'    '. ;  |   ||   ||---'|   ||   ||   ||   |
 \.'        '/  `---'|`--'`---'`   '`   '`   '`---'
  '.       .'        |    empowering the smart home
    '-----'               3.0.0-SNAPHOT Build #1919
```

```

    .-"""-.                             _   _     _     ____  
  .`       `.    ___   ___   ___   ___ | | | |   / \   | __ ) 
 /    .''.   \  / _ \ / _ \ / _ \ / _ \| |_| |  / _ \  |  _ \ 
 ;  .'    '. ; | (_) | (_) |  __/| | | |  _  | / ___ \ | |_) |
 \.'        '/  \___/|  __/ \___/|_| |_|_| |_|/_/   \_\|____/ 
  '.       .'        |_|      empowering the smart home
    '-----'                   3.0.0-SNAPHOT Build #1919
```

```
    _....._
  .' _..._ '.                            _   _     _     ____  
 / /`     `\ \    ___   ___   ___   ___ | | | |   / \   | __ ) 
; ;   .''.  ; ;  / _ \ / _ \ / _ \ / _ \| |_| |  / _ \  |  _ \ 
| | .'.''.'.| | | (_) | (_) |  __/| | | |  _  | / ___ \ | |_) |
; .'.'    '.; ;  \___/|  __/ \___/|_| |_|_| |_|/_/   \_\|____/ 
 '.'.     .' /        |_|      empowering the smart home
  '._`"'"`_.'                  3.0.0-SNAPHOT Build #1919
     `````    
```

Try to imagine them with colors, it might help ;)
Anyways let me know if there's one of them that could make it, I don't plan on working on those a lot ;)

**Comment by kaikreuzer** [comment]
Neat idea, but to be honest, they all look a bit awkward. Especially our logo does not seem to go well as ASCII art, so I'd tend to leave it out and stick to the "openHAB" text only.
Wrt the "empowering the smart home" slogan, the nice alignment with the version and build info only worked because you left out an "S" in SNAPHOT... I'd also tend to leave out the slogan. So replacing the font of "openHAB" makes sense and I think that looks quite decent as well.
So to summarize, my suggestion would be the following:

```
                          _   _     _     ____  
   ___   ___   ___   ___ | | | |   / \   | __ ) 
  / _ \ / _ \ / _ \ / _ \| |_| |  / _ \  |  _ \ 
 | (_) | (_) |  __/| | | |  _  | / ___ \ | |_) )
  \___/|  __/ \___/|_| |_|_| |_|/_/   \_\|____/ 
        |_|         3.0.0-SNAPSHOT - Build #1919
```

**Comment by cweitkamp** [comment]
I like the non-italic font. It goes better with the current logo.

**Comment by ghys** [comment]
> they all look a bit awkward. Especially our logo does not seem to go well as ASCII art

True... turns out circles and diagonal lines aren't so easy to do in ASCII 🤷‍♂️ did my best but not satisfied with the result either.

> you left out an "S" in SNAPHOT... 

😲🤦

I'll add the colors and submit a PR with the straight font.

**Comment by mstormi** [comment]
anyone care to add an "ian" so I can use it as the welcome banner for openHABian, too ?

**Comment by kaikreuzer** [comment]
That should hopefully be an easy exercise for @ghys!

**Comment by mstormi** [comment]
yeah, but @ecdye was faster: https://github.com/openhab/openhabian/pull/1184
